api/config/sync-catalog/32-sync-catalog-cronjob.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
folder name `sync-catalog` -> `catalog refresh`? to maintain catalog refresh as consistent everywhere? WDYT?